modules/ROOT/pages/about-api-catalog-cli.adoc

```diff
@@ -0,0 +1,689 @@
+                ]) { 
+                    sh 'api-catalog-cli publish-asset -d $CATALOG_DESCRIPTOR -o $ANYPOINT_ORG_ID -a https://anypoint.mulesoft.com/ --trigger=branch:main' <2>
+                }
+            }
+        }
+         stage('Deploy') {    
+            steps {
+                // Any deployment tasks to be performed, here.
+            }
+        }
+    }
+----
+<1> Defines the organization and descriptor file values
+<2> Runs the `api-catalog publish-asset` command to publish APIs found in the `main` Github branch
+
+== Authentication
+
+You can configure Anypoint CLI authentication with username and password, client ID and client secret, or a bearer token.
+At least one method is required.
+
+=== Username and Password
+
+If you don't log in using single sign-on (SSO), you can use your username and password to log into the CLI directly.
+
+[%header%autowidth.spread,cols="a,a"]
+|===
+| Argument | Description
+| `username` | Your Anypoint Platform username
+
+You can also pass this value using the environment variable `export ANYPOINT_USERNAME=<name>`.
+| `password` | Your Anypoint Platform password
+
+You can also pass this value using the environment variable `export ANYPOINT_PASSWORD=<pwd>`.
+|===
+
+For information about logging in using SSO, see xref:access-management::external-identity.adoc[About Identity Management].
+
+=== Client ID and Client Secret
+
+You can configure a Connected App with the `client_credentials` grant type to use with the CLI.
+
+[%header%autowidth.spread,cols="a,a"]
+|===
+| Argument | Description
+| `client_id` | The ID of the Connected App
+
+You can also pass this value using the environment variable `export ANYPOINT_CLIENT_ID=<client_id>`.
+| `client_secret` | The client secret of the Connected App
+
+You can also pass this value using the environment variable `export ANYPOINT_CLIENT_SECRET=<client_secret>`.
+|===
+
+For more information about Connected Apps, see xref:access-management::connected-apps-overview.adoc[Connected Apps].
+
+=== Environment Variable Usage
+
+Keep the following in mind while using API Catalog CLI:
+
+* Command-line options override environment variables. +
+If you don't pass a command-line Argument, the default profile properties are used.
+* If not specified, the default environment is production.
+
+=== Timeout
+
+Your Anypoint session expires based on the *Default session timeout* configured in your Organization settings.
+
+// Authentication end 
+
+== Create a Descriptor File Manually
+
+You can optionally create a descriptor file manually rather than using the `api-catalog create-descriptor` command to create it. 
+
+After you create the descriptor file, copy it to the file path or repository where your API projects are stored. 
+
+The descriptor file must start with the following:
+
+---
+
+ #%Catalog Descriptor 0.6
+
+---
+
+TIP: Run the `api-catalog publish-asset` command using the `--dry-run` option to ensure your manually-created descriptor file is valid. 
```

Comment:
Yeah...I'm not convinced that we need to use notice formats for NOTES and TIPS that could just as easily be part of the narrative...unless you consider this a deliverable that is more for scanning than otherwise.

Reply:
I think of every document as something for scanning, especially nowadays. But I'm open to making it part of the text.